documentation/plain_rules.md

```diff
@@ -213,3 +214,64 @@ Updates the status of a twitter account, with the text build from the `template`
 
 The `template` field performs [string substitution](#string-substitution-syntax).
 
+## Metadata
+
+Metadata values can be accessed by adding the suffix `__metadata__x` to the attribute name, being `x` the name of the metadata attribute. This name can be used in the EPL text of the rule and in the parameters of the action which accept string substitution.
```

Comment:
In addition, looking to the example, additional `__` "tokens" can be used to navigate in the attribute value, assuming this is a compound, e.g. `__metadata__crs__system`. That should be also mentioned in the documentation.